docs/source/repair/repair-slower.rst

```diff
@@ -5,9 +5,8 @@ Repair slower
 You can make repair run slower by changing the level of parallelism or intensity.
 By default Scylla Manager runs repairs with full parallelism and intensity one.
 
-Reduce amount of work performed by repair master node.
-Try setting ``--intensity 0.5``, that reduces the number of token ranges repaired by a repair master node in a single chunk of work.
-You can further decrease it to almost 0 forcing Scylla to repair a single token range at a time.
+Reduce amount of work performed by a single Scylla repair job.
+If ``--intensity`` has been previously increased, try setting it to ``--intensity 1``. That reduces the number of token ranges repaired in a single chunk of work.
```

Comment:
Better stick to the same terminology across the whole document: s/chunk of work/repair job/